tasks.py

```diff
@@ -0,0 +1,646 @@
+        )
+        run(
+            "test -d /home/linuxbrew/.linuxbrew && eval \$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)",
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            'test -r ~/.bash_profile && echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.bash_profile',
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            'echo "eval $($(brew --prefix)/bin/brew shellenv)"',
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            "sudo apt-get install build-essential curl git", hide=not verbose, warn=True
+        )
+        if run("command -v brew", hide=not verbose, warn=True).ok:
+            run('echo "===>brew {}"'.format(MESSAGE_OK))
+        else:
+            run('echo "===>brew {}"'.format(MESSAGE_FAILED))
+
+    if PLATFORM == "Darwin":
+        if run("command -v cask ", hide=not verbose, warn=True).ok:
+            run('echo "===>Cask {}"'.format(MESSAGE_ALREADY_INSTALLED))
+        else:
+            if run("brew install cask").ok:
+                run('echo "===>Cask {}"'.format(MESSAGE_OK))
+            else:
+                run('echo "===>Cask {}"'.format(MESSAGE_FAILED))
+
+    # creating a local setting file
+    if run('cd project/settings && touch local.py && echo "DEBUG=True" > local.py').ok:
+        run('echo "===>Successfully configured settings for local development"')
+
+
+@task
+def install_dependencies(c, verbose=False):
+    """
+
+    Install all the dependencies of the API
+    usage: invoke install-dependencies or invoke install-dependencies --verbose
+
+    """
+
+    run("echo '***INSTALLING ALL DEPENDENCIES***'")
+    packages = ("", "django-sslserver")
+    for package in packages:
+
+        if run("pipenv install {}".format(package), hide=not verbose, warn=True).ok:
+            run('echo "===>{} {}"'.format(package, MESSAGE_OK))
+        else:
+            run('echo "===>{} {}"'.format(package, MESSAGE_FAILED))
+
+
+@task
+def pygraphviz(c, verbose=False):
+    """
+
+    Install graphyviz & pygraphviz 
+    usage: invoke pygraphviz or invoke pygraphviz --verbose
+
+    """
+
+    if PLATFORM == "Linux":
+        # Installing graphviz
+        if run(
+            "sudo apt-get update && sudo apt-get install graphviz",
+            hide=not verbose,
+            warn=True,
+        ).ok:
+            run('echo "Graphyviz {}"'.format(MESSAGE_OK))
+        else:
+            run('echo "Graphyviz {}"'.format(MESSAGE_FAILED))
+        # Installing pygraphviz
+        if run(
+            "sudo pip3 install pygraphviz --install-option='--include-path=/usr/include/graphviz' --install-option='--library-path=/usr/lib/graphviz/'",
+            hide=not verbose,
+            warn=True,
+        ).ok:
+            run('echo "===>pygraphyviz {}"'.format(MESSAGE_OK))
+        else:
+            run('echo "===>pygraphyviz {}"'.format(MESSAGE_FAILED))
+
+    elif PLATFORM == "Darwin":
```

Comment:
leave a comment detailing why we're targeting different library and include paths. I'm guessing this has to do with how mac OS X deals with the underlying Graphviz dependency (the C lib), but it might help future readers to be explicit as to why we're doing things this way.

Reply:
Can you explain to me the reason as welll? cz I really do not know why we must provide the path to Graphviz in the installation process. I know that pygraphviz allows you to manipulate stuff made by Graphviz in python. But I am not sure why exactly we must provide the path.

Reply:
http://pygraphviz.github.io/documentation/pygraphviz-1.5/install.html#providing-path-to-graphviz

```We tried our best to discover graphviz location automatically, but if you would like specify specific location for graphviz you may provide additrional parameters to specify graphviz location```